lib/view/widget/button/pictron_base_button.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+
+class PictronBaseButton extends ButtonBar {
+  PictronBaseButton({
+    Key key,
+    String text = "",
+    Function function,
+    TextStyle textStyle,
+  }) : super(
+          children: <Widget>[
+            RaisedButton(
+                onPressed: () {
+                  function();
+                },
```

Comment:
Maybe there is a better way to express this:

```dart
    children: <Widget>[
            RaisedButton(onPressed: function)
    ...
```

Lets try to create self descriptive variables. The name "function" is not as descriptive as "onPressed".